### ISSUE TITLE
Can't remove or rename tests

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-practices#1444]
Review [comments about test merge conflicts in 1356](https://github.com/w3c/aria-practices/pull/1356#issuecomment-655694958).

We seem to get caught in a loop when trying to remove a test file. PR 1356 originally moved the editor menubar example up one level in the directory structure and renamed it. Jon tried to adjust the tests in the same PR, but doing so either resulted in a state where there were merge conflicts or a state where tests failed.

Our current work around is:
1. Instead of move/rename of the example, PR 1356 now creates a new version, leaving existing the same.
2. After we merge the new, make a PR to delete the old.

Is this how it must be done to get through CI? When PR 1356 was first created, it did a move/rename and the tests were passing and there were no merge conflicts. So, it does seem like has something has changed, but we don't know what.

**Comment by spectranaut** [comment]
I looked into this for a moment, but it's really hard to follow a long, because there are so many adds and deletes and restarted builds. Can we try deleting the renamed files in #1359 so I can see what the failure is there, and if it's repeatable or something unique to #1356?

**Comment by a11ydoer** [comment]
> I looked into this for a moment, but it's really hard to follow a long, because there are so many adds and deletes and restarted builds. Can we try deleting the renamed files in #1359 so I can see what the failure is there, and if it's repeatable or something unique to #1356?

This method may also help to check correct folder structure relationship with testing file.
